bundles.md

```diff
@@ -0,0 +1,133 @@
+# Attestation Bundle
+
+An attestation bundle is a collection of multiple attestations in a single file.
+This allows attestations from multiple different points in the software supply
+chain (e.g. Provenance, Code Review, Test Result, vuln scan, ...) to be grouped
+together, allowing users to make decisions based on all available information.
+
+**NOTE**: Attestation Bundles themselves are **not authenticated** instead each
+individual attestation is authenticated
+([using DSSE](https://github.com/secure-systems-lab/dsse)). As such, an attacker
+might be able to _delete_ an attestation without being detected.  Predicates that
+follow [the monotonic principle](spec/README.md#parsing-rules) should not have any
+issues with this behavior.
+
+## Data structure
+
+Attestation Bundles use [JSON Lines](https://jsonlines.org/) to store multiple
+[DSSEs](https://github.com/secure-systems-lab/dsse).
+
+*  Each line MUST contain a _single_ DSSE.
+*  Each line MAY contain any DSSE `payloadType`
+*  in-toto attestations (`payloadType` == `application/vnd.in-toto+json`) MAY use
+   any ` _type`/`predicateType`
+*  [in-toto Statements](spec/README.md#statement)
+   in a Bundle MAY reference different Subjects
```

Comment:
What are Subjects again?

Reply:
A subject is the "Set of software artifacts that the attestation applies to. Each element represents a single software artifact." [from here](https://github.com/in-toto/attestation/blob/main/spec/README.md#statement)